host/CMakeLists.txt

```diff
@@ -494,6 +516,11 @@ endif ()
 install(
   TARGETS oehost
   EXPORT openenclave-targets
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/host)
+
+install(
+  TARGETS oehostverify
```

Comment:
Question: Do we still need the openenclave-targets as part of OEHOSTVERIFY (line 527 - 531)?

Reply:
Good point. I'll need to test it out.